### PR TITLE
Enable to copy raw svg

### PR DESCRIPTION
**Pull request by daniarsuprianto**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -3,25 +3,56 @@ body {
...
+}
+
+.icon svg {
+  width: 48px;
+  height: 48px;
+  cursor: pointer;
+}
+
+.icon p {
+  font-size: 12px;
```
Comment (larsdouweschuitema) [inline src/index.css:57]: Mixing px and rem

File: src/index.js [hunk 2]
```diff
@@ -28,5 +28,15 @@ var app = new Vue({
           (error) => `An error occurred while loading the component: ${error}`
         );
     },
+    copy(icon) {
+      navigator.clipboard.writeText(icon.svg).then(
```
Comment (larsdouweschuitema) [inline src/index.js:32]: Using async/await is more maintainable and elegant.

**Review by larsdouweschuitema — APPROVED** [review]

**Comment by larsdouweschuitema** [comment]
Approved the PR. Added two comments, but i'll leave it up to you to change them. One is more a preference, the other is very small 👍

File: src/index.js [hunk 3]
```diff
@@ -28,5 +28,13 @@ var app = new Vue({
           (error) => `An error occurred while loading the component: ${error}`
         );
     },
+    async copy(icon) {
+      try {
+        await navigator.clipboard.writeText(icon.svg);
+        alert(`"${icon.title}" svg is copied to your clipboard`);
```
Comment (sukalas-a) [inline src/index.js:34]: Why are you using alerts?

Comment (daniarsuprianto) [reply src/index.js:34]: to confirm that it's copied, and because it's easy rather than creating custom alert component 🤪

Comment (sukalas-a) [reply src/index.js:34]: Fair enough, I just have ptsd WITH alert deadlocks.

Comment (larsdouweschuitema) [reply src/index.js:34]: The one moment, where an alert is useful!

**Review by sukalas-a — APPROVED** [review]

**Review by larsdouweschuitema — APPROVED** [review]

**Pull request merged**